src/controllers/recipes_controller.js

```diff
@@ -71,9 +72,42 @@ module.exports.processRecipesJson = (req, res) => {
 
     pythonProcess.stdout.on("end", () => {
         res.status(200).json(dataString.toString());
-    });
+    });    
 };
 
+module.exports.planMeals = ({body: {recipeIds = null}}, res) => {
+    const recipeIdQueries = recipeIds.map(recipeId => (callback) => recipesModel.selectRecipeById(recipeId, callback))
+
+    async.parallel(recipeIdQueries, (err, results) => {
+        const nutritions = {
+            calories: {amount: 0, unit: null},
+            fat: {amount: 0, unit: null},
+            cholesterol: {amount: 0, unit: null},
+            carbohydrates: {amount: 0, unit: null},
+            protein: {amount: 0, unit: null},
+            sugars: {amount: 0, unit: null},
+        }; 
+
+        results.forEach(({nutrition: {calories, fat, cholesterol, carbohydrates, protein, sugars}}) => {
+            if (nutritions.calories.unit == null){
+                nutritions.calories.unit = calories.unit
+                nutritions.fat.unit = fat.unit
+                nutritions.cholesterol.unit = cholesterol.unit
+                nutritions.carbohydrates.unit = carbohydrates.unit
+                nutritions.protein.unit = protein.unit
+                nutritions.sugars.unit = sugars.unit
+            }
+
+            nutritions.calories.amount += calories.amount
+            nutritions.fat.amount += fat.amount
+            nutritions.cholesterol.amount += cholesterol.amount
+            nutritions.carbohydrates.amount += carbohydrates.amount
+            nutritions.protein.amount += protein.amount
+            nutritions.sugars.amount += sugars.amount
+        });
```

Comment:
Would there be any cases where the units would be different? (ie. grams and kilograms)

Reply:
Can you make an issue for that and link it here, so we dont forget :D